bin/cheribsdtest/cheribsdtest.c

```diff
@@ -86,6 +88,7 @@ static const struct cheri_test *running_test;
 
 static int tests_run, tests_skipped;
 static int tests_failed, tests_passed, tests_xfailed, tests_xpassed;
+static int execed;
```

Comment:
This isn't a great name; cheribsdtest is always exec'ed, the point is it's being told to run a test's child function.

Reply:
That poor choice was mine. Maybe `reexeced` or `is_execed_child` instead? (Should probably be a `bool` now that I think of it.)

Reply:
The other flags like coredump_enabled and debugger_enabled are still `int`s so I believe `is_execed_child` could stay as an `int`.

Reply:
Sounds fine, we can follow up with a switch to bools later